From 46d11f0f9af9134c7f87b8b9c7f106d6869565ba Mon Sep 17 00:00:00 2001
From: "Liu, Kai1" <kai1.liu@intel.com>
Date: Tue, 6 Aug 2024 09:45:23 +0800
Subject: [PATCH] Reduce POLL_INTERVAL_MS to 50ms to optimize cold boot

Reduce POLL_INTERVAL_MS to 50ms and increase POLL_COUNT to 100 to
optimize cold boot.

Tracked-On: OAM-123070
Signed-off-by: Liu, Kai1 <kai1.liu@intel.com>
---
 src/com/android/providers/media/fuse/FuseDaemon.java | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/com/android/providers/media/fuse/FuseDaemon.java b/src/com/android/providers/media/fuse/FuseDaemon.java
index 34836af41..ee996e13f 100644
--- a/src/com/android/providers/media/fuse/FuseDaemon.java
+++ b/src/com/android/providers/media/fuse/FuseDaemon.java
@@ -36,8 +36,8 @@ import java.util.Objects;
  */
 public final class FuseDaemon extends Thread {
     public static final String TAG = "FuseDaemonThread";
-    private static final int POLL_INTERVAL_MS = 1000;
-    private static final int POLL_COUNT = 5;
+    private static final int POLL_INTERVAL_MS = 50;
+    private static final int POLL_COUNT = 100;
 
     private final Object mLock = new Object();
     private final MediaProvider mMediaProvider;
-- 
2.34.1

